cmd/snapctl/commands.go

```diff
@@ -100,7 +100,7 @@ var (
 				},
 				{
 					Name:   "unload",
-					Usage:  "unload",
+					Usage:  "unload <plugin_type>:<plugin_name:<plugin_version> or unload -t <plugin_type> -n <plugin_version> -v <plugin_version>",
```

Comment:
Typo in this line with plugin_name, should be <plugin_name>, currently <plugin_name